### PR TITLE
Update de.js

### PR DESCRIPTION
**Pull request by Customsexe**
Last one i promise xDD. I always find mistakes c.c (IGN Customs_KR)

### CODE REVIEW

**Review by lightversem — APPROVED** [review]

File: de.js [hunk 1]
```diff
@@ -2,15 +2,15 @@ module.exports = `
...
+menu.btn.maps Karten
 menu.btn.mods Mods
 menu.btn.settings Einstellungen
 menu.btn.customize Anpassen
 menu.btn.invite Einladen
 menu.btn.invite.clicked Link kopiert
 menu.btn.join Beitreten
 menu.btn.host Spiel veranstalten
-menu.btn.browser Server Browser
+menu.btn.browser Server Suchmaschine
```
Comment (Sqlele) [inline de.js:13]: "Suchmaschine" doesn't really fit as a replacement to "servers"

File: de.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ generic.load Laden
 generic.search Suchen
 generic.leave Verlassen
 generic.default Voreinstellung
-generic.submit Senden
+generic.submit Einreichen
```
Comment (Sqlele) [inline de.js:25]: "Einreichen" sounds awkward

File: de.js [hunk 3]
```diff
@@ -57,32 +57,32 @@ generic.username Benutzername eingeben
...
 generic.register Registrieren
 generic.login Einloggen
-generic.accuracy Genauigkeit
+generic.accuracy Präzision
 player.waiting Warten auf Spieler
 player.died Du bist gestorben
 player.killed Getötet von
-player.killstreak {0} auf einer {1} Abschussserie
-player.inactive Wegen Inaktivität entfernt
+player.killstreak {0} auf einer {1} Abschusssträhne
```
Comment (Sqlele) [inline de.js:64]: "Abschusssträhne" sounds awkward

File: de.js [hunk 4]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
 settings.interface.hitInd Hitmarker anzeigen
 settings.interface.hitColor Hitmarker Farbe
 settings.interface.dmg Schaden anzeigen
-settings.interface.dmgType Show Damage
-settings.interface.dmgType.default Rounded
+settings.interface.dmgType Schaden anzeigen
+settings.interface.dmgType.default Gerunded
```
Comment (Sqlele) [inline de.js:176]: Spelling error, its "Gerundet"

File: de.js [hunk 5]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.dmgType.float Float
 settings.interface.dmgCol Schadens-Farbe
 settings.interface.critCol Kritischer Treffer Farbe
 settings.interface.header Schnittstelle
 settings.interface.scale Benutzeroberfläche skalieren
-settings.interface.theme Spieldesign
+settings.interface.theme Spielgestaltung
```
Comment (Sqlele) [inline de.js:182]: Sounds okay but I would prefer "Spieldesign"

File: de.js [hunk 6]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.ui Benutzeroberfläche anzeigen
 settings.interface.scoreboard Alte Anzeigetafel
 settings.interface.oldBrowser Alter Server Browser
 settings.interface.rarityanim Seltenheits-Animationen ausschalten
 settings.interface.showPfp Profilbilder anzeigen
-settings.interface.showTradeMsg Trade-Nachrichten anzeigen
+settings.interface.showTradeMsg Handels-Nachrichten anzeigen
 settings.interface.hideNonTrade Verstecke nicht handelbare Skins
-settings.interface.hp Dynamische HP Balken
+settings.interface.hp Dynamische Lebens Balken
```
Comment (Sqlele) [inline de.js:190]: I would leave it as "HP" becaue it doesn't really fit with line 221 and 222

File: de.js [hunk 7]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.dmgScale Schadens Größe
-settings.interface.killf Kill Feed anzeigen
-settings.interface.ping Ping anzeigen
+settings.interface.killf Abschuss Feed anzeigen
+settings.interface.ping Verbindung anzeigen
```
Comment (Sqlele) [inline de.js:193]: "Ping" and "Verbindung" are two completely different things

File: de.js [hunk 8]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.net-stats Netzwerkstatistiken anzeigen
 settings.interface.fps FPS-Rate anzeigen
 settings.interface.deaths Tode anzeigen
 settings.interface.kd K/D Zähler anzeigen
-settings.interface.kills Abschussserie anzeigen
+settings.interface.kills Abschusssträhne anzeigen
```
Comment (Sqlele) [inline de.js:198]: Same thing as in line 64

File: de.js [hunk 9]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
-settings.interface.speedOffY Geschwindigkeits Y Offset
+settings.interface.speedOffX Geschwindigkeits X Versetzung
+settings.interface.speedOffY Geschwindigkeits Y Versetzung
 settings.interface.speedOpacity Geschwindigkeits Deckkraft
 settings.interface.speedScale Geschwindigkeits Größe
 settings.interface.speedColor Geschwindigkeits Farbe (Momentan)
 settings.interface.speedColorMax Geschwindigkeits Farbe (Max)
 settings.interface.hideNames Namens-Anzeige
-settings.interface.hideNames.team Nur Team
+settings.interface.hideNames.team Nur Mannschaft
```
Comment (Sqlele) [inline de.js:210]: I would leave it as "Team"

File: de.js [hunk 10]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.hideNames.enemy Nur Gegner
 settings.interface.hideNames.all Jeder
 settings.interface.nametagStyle Namensanzeige-Style
 settings.interface.nametagOpac Namensanzeige Dreckkraft
-settings.interface.nametagScale Nametag Skalierung
-settings.interface.nametagStyle.health Nur HP
+settings.interface.nametagScale Namensanzeige Skalierung
+settings.interface.nametagStyle.health Nur Leben
```
Comment (Sqlele) [inline de.js:216]: Same thing as in line 190

File: de.js [hunk 11]
```diff
@@ -172,69 +172,69 @@ settings.post.ssaoRadius SSAO Radius
...
 settings.interface.nametagStyle.all Alles
 settings.interface.healthNum Nametag Lebens-Nummer
-settings.interface.healthColT Nametag Team-HP-Farbe
-settings.interface.healthColE Nametag Gegner-HP-Farbe
-settings.interface.xpBarType XP Bar Type
-settings.interface.xpBarCol XP Leisten-Farbe
-settings.interface.xpBarCol2 XP Bar Color 2
-settings.interface.xpBarOpac XP Leisten-Deckkraft
-settings.interface.adsObjOpac ADS Objective Opacity
+settings.interface.healthColT Nametag Mannschafts-Lebens-Farbe
```
Comment (Sqlele) [inline de.js:221]: Same thing as in line 210

File: de.js [hunk 12]
```diff
@@ -286,7 +286,7 @@ settings.crosshairm.opacity.use benutzerdefinierte Deckkraft benutzen
 settings.crosshairm.opacity.custom benutzerdefinierte Deckkraft
 settings.hitmarker.header Hitmarker
 settings.hitmarker.color Farbe
-settings.hitmarker.kcolor Kill Farbe
+settings.hitmarker.kcolor Eleminierungs Farbe
```
Comment (Sqlele) [inline de.js:289]: Spelling error, it's "Eliminierung"

File: de.js [hunk 13]
```diff
@@ -301,8 +301,8 @@ settings.onKillS.style.0 Voreinstellung
 settings.onKillS.style.1 Auflösung reduzieren
 settings.onKillS.style.2 Sichtfeld reduzieren
 settings.onKillS.style.3 Sichfeld erhöhen
-settings.serverb.header Server Browser
-settings.serverb.oldBrowser Klassischer Browser
+settings.serverb.header Server Suchmaschine
+settings.serverb.oldBrowser Klassische Suchmaschine
```
Comment (Sqlele) [inline de.js:305]: Same thing as in line 304

File: de.js [hunk 14]
```diff
@@ -301,8 +301,8 @@ settings.onKillS.style.0 Voreinstellung
 settings.onKillS.style.1 Auflösung reduzieren
 settings.onKillS.style.2 Sichtfeld reduzieren
 settings.onKillS.style.3 Sichfeld erhöhen
-settings.serverb.header Server Browser
-settings.serverb.oldBrowser Klassischer Browser
+settings.serverb.header Server Suchmaschine
```
Comment (Sqlele) [inline de.js:304]: Same thing as in line 13

File: de.js [hunk 15]
```diff
@@ -362,7 +362,7 @@ settings.viewmodel.vmSetts Ansichtseinstellungen
 settings.viewmodel.fov Sichtfeld
 settings.viewmodel.fps Waffen Sichtfeld
 settings.viewmodel.bobbing Waffen Bewegung
-settings.viewmodel.leaning Waffen Lehnen
+settings.viewmodel.leaning Waffen Neigung
```
Comment (Sqlele) [inline de.js:365]: I wouldn't choose "Neigung" or "Lehnen", I would use "Lehnung" instead

File: de.js [hunk 16]
```diff
@@ -373,55 +373,55 @@ settings.viewmodel.show.hands Hände anzeigen
...
 settings.viewmodel.weapSwapY Waffenwechsel Y
 settings.viewmodel.weapReloadY Waffen-Nachlade Y
-settings.viewmodel.yOffADS Waffen Zielen Y Offset
+settings.viewmodel.yOffADS Waffen Zielen Y Versetzung
 settings.viewmodel.adsFovMlt Zielen FOV Power
 settings.viewmodel.toggleADS Toggle Weapon ADS
-settings.viewmodel.wallLean Wall Slide/Jump Lean
-settings.viewmodel.wallLeanMlt Wall Slide/Jump Lean
+settings.viewmodel.wallLean Wall Rutsch/Sprung Neigung
+settings.viewmodel.wallLeanMlt Wand Rutsch/Sprung Neigung
```
Comment (Sqlele) [inline de.js:385]: "Wand Rutsch" sounds very awkward and I wouldn't use it

File: de.js [hunk 17]
```diff
@@ -373,55 +373,55 @@ settings.viewmodel.show.hands Hände anzeigen
...
 settings.editing.header Bearbeiten
-settings.editing.health.high HUD HP Hoch
-settings.editing.health.low HUD HP Niedrig
+settings.editing.health.high HUD Leben Hoch
+settings.editing.health.low HUD Leben Niedrig
```
Comment (Sqlele) [inline de.js:388]: Same thing as in line 190 #

File: de.js [hunk 18]
```diff
@@ -373,55 +373,55 @@ settings.viewmodel.show.hands Hände anzeigen
...
 settings.viewmodel.adsFovMlt Zielen FOV Power
 settings.viewmodel.toggleADS Toggle Weapon ADS
-settings.viewmodel.wallLean Wall Slide/Jump Lean
-settings.viewmodel.wallLeanMlt Wall Slide/Jump Lean
+settings.viewmodel.wallLean Wall Rutsch/Sprung Neigung
+settings.viewmodel.wallLeanMlt Wand Rutsch/Sprung Neigung
 settings.editing.header Bearbeiten
-settings.editing.health.high HUD HP Hoch
-settings.editing.health.low HUD HP Niedrig
+settings.editing.health.high HUD Leben Hoch
```
Comment (Sqlele) [inline de.js:387]: Same thing as in line 190

File: de.js [hunk 19]
```diff
@@ -373,55 +373,55 @@ settings.viewmodel.show.hands Hände anzeigen
...
+settings.editing.progmOffX Auflade-Anzeige X Versetzung
+settings.editing.progmOffY Auflade-Anzeige Y Versetzung
 settings.editing.progmRot Auflade-Anzeige Rotation
 settings.editing.saturation Farbsättigung (Im Spiel)
 settings.editing.menuSaturation Farbsättigung (Benutzeroberfläche)
 settings.editing.hue Farbton (Im Spiel)
 settings.editing.menuHue Farbton (Benutzeroberfläche)
 settings.editing.vignette Vignette
-settings.editing.feedLimit Killfeed Limit
+settings.editing.feedLimit Abschuss feed Limit
```
Comment (Sqlele) [inline de.js:417]: Remove the space between "Abschuss" and "feed"

File: de.js [hunk 20]
```diff
@@ -448,10 +448,10 @@ settings.mods.kills Abschuss Symbol Bild
 settings.mods.kills.paste Abschuss Symbol Bild URL
 settings.mods.deaths Tode Symbol Bild
 settings.mods.deaths.paste Tode Symbol Bild URL
-settings.mods.streak Abschussserie Symbol Bild
-settings.mods.streak.paste Abschussserie Symbol Bild URL
-settings.mods.timer Timer Symbol Bild
-settings.mods.timer.paste Timer Symbol Bild URL
+settings.mods.streak Abschusssträhnen Symbol Bild
```
Comment (Sqlele) [inline de.js:451]: Same thing as in line 64

File: de.js [hunk 21]
```diff
@@ -448,10 +448,10 @@ settings.mods.kills Abschuss Symbol Bild
...
+settings.mods.streak.paste Abschusssträhnem Symbol Bild URL
```
Comment (Sqlele) [inline de.js:452]: Same thing as in line 64

File: de.js [hunk 22]
```diff
@@ -448,10 +448,10 @@ settings.mods.kills Abschuss Symbol Bild
...
+settings.mods.timer Zeit Symbol Bild
```
Comment (Sqlele) [inline de.js:453]: "Zeit" isn't bad but it sounds awkward #

File: de.js [hunk 23]
```diff
@@ -448,10 +448,10 @@ settings.mods.kills Abschuss Symbol Bild
...
+settings.mods.timer.paste Zeit Symbol Bild URL
```
Comment (Sqlele) [inline de.js:454]: Same thing as in line 453

File: de.js [hunk 24]
```diff
@@ -491,10 +491,10 @@ windows.loadout.color.hair Haarfarbe
 windows.loadout.dye Körperfarbe
 windows.loadout.pet Haustier
 windows.loadout.wrist Handgelenk
-windows.loadout.stat Killcard Statistiken
+windows.loadout.stat Abschusscard Statistiken
 windows.loadout.flag Länderflagge
 windows.loadout.badge Benutzerdefiniertes Abzeichen
-windows.loadout.streaks Streaks
+windows.loadout.streaks Strähnen
```
Comment (Sqlele) [inline de.js:497]: Same thing as in line 64

File: de.js [hunk 25]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
 windows.controls.confirm Interaktion bestätigen
 windows.controls.drop Fallen lassen
 windows.controls.wepVis Waffen Sichtbarkeit umschalten
-windows.controls.kickVoteY Vote Kick [Ja]
-windows.controls.kickVoteN Vote Kick [Nein]
-windows.controls.kpdVoteY KPD Vote [Ja]
-windows.controls.kpdVoteN KPD Vote [Nein]
+windows.controls.kickVoteY Ausschlussabstimmung [Ja]
```
Comment (Sqlele) [inline de.js:537]: "Ausschlussabstimmung" sounds awkward

File: de.js [hunk 26]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
+windows.controls.kickVoteN Ausschlussabstimmung [Nein]
```
Comment (Sqlele) [inline de.js:538]: Same thing as in line 537

File: de.js [hunk 27]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
 windows.controls.specFirst First Person umschalten
 windows.controls.specNames Erzwungene Namen umschalten
 windows.controls.specFocus Spielerfokus umschalten
 windows.controls.hidePlayers Verstecke Spieler
-windows.controls.streak0 Abschussserie 1
-windows.controls.streak1 Abschussserie 2
-windows.controls.streak2 Abschussserie 3
-windows.controls.streak3 Abschussserie 4
-windows.controls.streak4 Abschussserie 5
+windows.controls.streak0 Abschusssträhne 1
```
Comment (Sqlele) [inline de.js:547]: Same thing as in line 64

File: de.js [hunk 28]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
+windows.controls.streak1 Abschusssträhne 2
+windows.controls.streak2 Abschusssträhne 3
```
Comment (Sqlele) [inline de.js:549]: Same thing as in line 64

File: de.js [hunk 29]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
+windows.controls.streak3 Abschusssträhne 4
```
Comment (Sqlele) [inline de.js:550]: Same thing as in line 64

File: de.js [hunk 30]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
+windows.controls.streak4 Abschusssträhne 5
```
Comment (Sqlele) [inline de.js:551]: Same thing as in line 64

File: de.js [hunk 31]
```diff
@@ -560,19 +560,19 @@ windows.controls.propRand Wähle zufällige Gestalt
 windows.controls.propRot Rotation umschalten
 windows.controls.propRotR Rotation zurücksetzen
 windows.controls.premium0 Noclip
-windows.controls.premium1 Godmode
+windows.controls.premium1 Gottmodus
```
Comment (Sqlele) [inline de.js:563]: "Gottmodus" isn't really a german word and sounds awkward as well

File: de.js [hunk 32]
```diff
@@ -560,19 +560,19 @@ windows.controls.propRand Wähle zufällige Gestalt
...
 windows.controls.premium2 Kill
 windows.controls.premium3 Smite
 windows.controls.reset Position zurücksetzen (Rennen & Parkour)
-windows.controls.resetLast Reset To Last Checkpoint (Parkour etc)
+windows.controls.resetLast Zurücksetzen zum letzen Speicherpunkt (Parkour etc)
```
Comment (Sqlele) [inline de.js:567]: Replace "Speicherpunkt" with "Kontrollpunkt" like you did in line 69

File: de.js [hunk 33]
```diff
@@ -560,19 +560,19 @@ windows.controls.propRand Wähle zufällige Gestalt
...
 windows.controls.pretend Spiel verstecken
 windows.controls.noclipSB Noclip
-windows.controls.godModeSB Godmode
+windows.controls.godModeSB Gottmodus
```
Comment (Sqlele) [inline de.js:570]: Same thing as in line 563

File: de.js [hunk 34]
```diff
@@ -696,16 +696,16 @@ matchmaker.GameFull Das Spiel ist voll.
 matchmaker.NoServersMatchQuery SPIEL WIRD AKTUALISIERT
 matchmaker.InvalidGameId Spiel wurde nicht gefunden.
 matchmaker.NoAvailableServers Server sind ausgelastet
-matchmaker.NoMatchingServers No Games Found<br><small>Try changing Quickplay options</small>
+matchmaker.NoMatchingServers Kein Spiel gefunden<br><small>Probiere veränderung der Quickplay Option</small>
 matchmaker.GameIssue Unbekannter Beitrittsfehler
-adblocker Deaktiviere deinen AdBlocker um gratis Krunkies zu erhalten.
+adblocker Deaktiviere deinen Werbungsblockierer um gratis Krunkies zu erhalten.
```
Comment (Sqlele) [inline de.js:701]: "Werbungsblockierer" sounds awkward and isn't really a german word

File: de.js [hunk 35]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
-server.config.nameTeam4 Team 4 Name
-server.config.nameTeam5 Team 5 Name
-server.config.t1Dmg Team 1 Schaden
-server.config.t2Dmg Team 2 Schaden
-server.config.t3Dmg Team 3 Schaden
-server.config.t4Dmg Team 4 Schaden
-server.config.t5Dmg Team 5 Schaden
-server.config.selTeam Team auswählen
-server.config.frFire Friendly Fire
+server.config.nameTeam1 Mannschaft 1 Name
```
Comment (Sqlele) [inline de.js:730]: Same thing as in line 210

File: de.js [hunk 36]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.nameTeam2 Mannschaft 2 Name
```
Comment (Sqlele) [inline de.js:731]: Same thing as in line 210

File: de.js [hunk 37]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.nameTeam3 Mannschaft 3 Name
+server.config.nameTeam4 Mannschaft 4 Name
```
Comment (Sqlele) [inline de.js:733]: Same thing as in line 210 @

File: de.js [hunk 38]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.nameTeam5 Mannschaft 5 Name
```
Comment (Sqlele) [inline de.js:734]: Same thing as in line 210

File: de.js [hunk 39]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.t1Dmg Mannschaft 1 Schaden
```
Comment (Sqlele) [inline de.js:735]: Same thing as in line 210

File: de.js [hunk 40]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.t2Dmg Mannschaft 2 Schaden
```
Comment (Sqlele) [inline de.js:736]: Same thing as in line 210

File: de.js [hunk 41]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.t3Dmg Mannschaft 3 Schaden
+server.config.t4Dmg Mannschaft 4 Schaden
```
Comment (Sqlele) [inline de.js:738]: Same thing as in line 210 @

File: de.js [hunk 42]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.t5Dmg Mannschaft 5 Schaden
```
Comment (Sqlele) [inline de.js:739]: Same thing as in line 210

File: de.js [hunk 43]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.selTeam Mannschaft auswählen
```
Comment (Sqlele) [inline de.js:740]: Same thing as in line 210

File: de.js [hunk 44]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
+server.config.frFire freundliches Feuer
```
Comment (Sqlele) [inline de.js:741]: "Freundliches Feuer" makes no sense

File: de.js [hunk 45]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
 server.config.allowSpect Zuschauen
-server.config.killRewards Abschuss-Belohnung
+server.config.killRewards Abschuss-Belohnungen
 server.config.headshotOnly Nur Kopfschüsse
 server.config.noSecondary Sekundärwaffen deaktivieren
-server.config.noStreaks Streaks deaktivieren
-server.config.canSlide Sliden
+server.config.noStreaks Abschusssträhnen deaktivieren
```
Comment (Sqlele) [inline de.js:746]: Same thing as in line 64

File: de.js [hunk 46]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
 server.config.gamRounds Runden
-server.config.objtvTime Objective-Zeit
+server.config.objtvTime Objekt-Zeit
 server.config.forceSpawn Automatische Respawn-Zeit
-server.config.intermTmr Intermission (s)
-server.config.scoreLimit Punkte Limit
-server.config.keepTScore Team Punktestand beibehalten
+server.config.intermTmr Pause (n)
+server.config.scoreLimit Punkte Limitierung
+server.config.keepTScore Mannschaft Punktestand beibehalten
```
Comment (Sqlele) [inline de.js:767]: Same thing as in line 210

File: de.js [hunk 47]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
 server.config.forceC Modus-Klassen erzwingen
 server.config.logTim Zeiten protokollieren (Rennen, Parkour)
-server.config.lstChkT Last Checkpoint Time Rewind (Race/Bhop)
+server.config.lstChkT Letze Speicherpunktzeit zurückspulen (Race/Bhop)
```
Comment (Sqlele) [inline de.js:770]: Spelling error, it's "Letzte"

File: de.js [hunk 48]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
 server.config.hitBoxPad Hitbox Polsterung
 server.config.hpRegen Lebens-Regeneration
 server.config.disableB Grenzen deaktivieren
 server.config.throwMel Werfbare Messer
 server.config.noCosm Kosmetische Gegenstände deaktivieren
 server.config.tstCmp Testmodus
 server.config.limitClasses Limitierte Klassen
-server.config.tmSize Teamgröße
+server.config.tmSize Mannschaftsgröße
```
Comment (Sqlele) [inline de.js:778]: Same as in line 210

File: de.js [hunk 49]
```diff
@@ -833,7 +833,7 @@ custom.host.ranked Im Ranked Modus kannst du kein Spiel hosten
 custom.host.wager Can't host match while in Wager.
 funds.insufficient Zu wenig Krunkies
 funds.out Sieht aus als hättest du keine Krunkies mehr.
-custom.passcode.long Passwort zu lang
+custom.passcode.long Passwort zu lange
```
Comment (Sqlele) [inline de.js:836]: "lang" is grammatically correct

File: de.js [hunk 50]
```diff
@@ -947,18 +947,18 @@ chall.name.dmg {0} Schaden zufügen {1}
 chall.name.bzk Erziele {0} Spielverderber-Abschüsse {1}
 chall.name.cc Erziele {0} hauchnahe Abschüsse {1}
 chall.name.rk Erziele {0} Vergeltungs-Abschüsse {1}
-event.doubleXP 2x Klassen XP in diesem Spiel aktiv
+event.doubleXP 2x Klassen EP in diesem Spiel aktiv
 ranked.join Ranked Spiel beigetreten. Das Verlassen eines Ranked Spiels wird mit einem temporären Bann bestraft
 ranked.progress Spiel in Gange
-ranked.team Tritt einem Team bei
+ranked.team Tritt einer Mannschaft bei
```
Comment (Sqlele) [inline de.js:953]: Same thing as in line 210

File: de.js [hunk 51]
```diff
@@ -947,18 +947,18 @@ chall.name.dmg {0} Schaden zufügen {1}
...
 ranked.switch Du kannst während der Runde deine Klasse nicht wechseln
 ranked.spectate Zuschauen ist im Ranked Modus nicht gestattet
 ranked.login Melde dich an um zu spawnen
 ranked.sync Sync Daten fehlen
-ranked.team.spawn Bitte tritt einem Team bei um zu spawnen
+ranked.team.spawn Bitte tritt einer Mannschaft bei um zu spawnen
```
Comment (Sqlele) [inline de.js:958]: Same thing as in line 210

File: de.js [hunk 52]
```diff
@@ -947,18 +947,18 @@ chall.name.dmg {0} Schaden zufügen {1}
...
 ranked.class.spawn Bitte wähle eine Klasse aus um zu spawnen
-modes.round.win Dein Team hat diese Runde gesichert
-modes.round.loss Dein Team kann diese Runde nicht mehr gewinnen
+modes.round.win Deine Mannschaft hat diese Runde gesichert
```
Comment (Sqlele) [inline de.js:960]: Same thing as in line 210

File: de.js [hunk 53]
```diff
@@ -947,18 +947,18 @@ chall.name.dmg {0} Schaden zufügen {1}
...
+modes.round.loss Deine Mannschaft kann diese Runde nicht mehr gewinnen
```
Comment (Sqlele) [inline de.js:961]: Same thing as in line 210

File: de.js [hunk 54]
```diff
@@ -999,15 +999,15 @@ commands.forfeit.later Versuche es in ein paar Sekunden nochmal
 commands.forfeit.voted Du hast schon abgestimmt
 commands.forfeit.limit Du kannst nur eine Abstimmung pro Spiel starten
 commands.forfeit.ended Spiel wird aufgegeben
-commands.forfeit.ended.other Das andere Team hat aufgegeben
+commands.forfeit.ended.other Die andere Mannschaft hat aufgegeben
```
Comment (Sqlele) [inline de.js:1002]: Same thing as in line 210

File: de.js [hunk 55]
```diff
@@ -999,15 +999,15 @@ commands.forfeit.later Versuche es in ein paar Sekunden nochmal
...
-server.message.votekicked {0} wurde per vote gekickt
+server.message.votekicked {0} wurde durch eine Abstimmung gekickt
 server.message.arrested {0} wurde vom K.P.D verhaftet
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
-event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
-event.easter.completed {0} completed the Egg hunt!
+event.easter.golden.spawn Ein goldenes Ei ist nahe!
+event.easter.golden.despawn Ein goldenes Ei ist weg!
```
Comment (Sqlele) [inline de.js:1010]: I would replace "ist weg" with "ist verschwunden"

File: de.js [hunk 56]
```diff
@@ -999,15 +999,15 @@ commands.forfeit.later Versuche es in ein paar Sekunden nochmal
...
+event.easter.found.new {0} fand {1}/{2} Eiern
```
Comment (Sqlele) [inline de.js:1011]: I would replace "fand {1}/{2} Eiern" with "hat {1}/{2} Eiern gefunden"

File: de.js [hunk 57]
```diff
@@ -999,15 +999,15 @@ commands.forfeit.later Versuche es in ein paar Sekunden nochmal
...
+event.easter.found.old Du hast dieses Ei schon gefunden.
+event.easter.completed {0} Schloss die Eiersuche ab!
```
Comment (Sqlele) [inline de.js:1013]: I would replace "Schloss die Eiersuche ab" with "hat die Eiersuche abgeschlossen"

File: de.js [hunk 58]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
-server.config.selTeam Team auswählen
-server.config.frFire Friendly Fire
+server.config.nameTeam1 Mannschaft 1 Name
+server.config.nameTeam2 Mannschaft 2 Name
+server.config.nameTeam3 Mannschaft 3 Name
+server.config.nameTeam4 Mannschaft 4 Name
+server.config.nameTeam5 Mannschaft 5 Name
+server.config.t1Dmg Mannschaft 1 Schaden
+server.config.t2Dmg Mannschaft 2 Schaden
+server.config.t3Dmg Mannschaft 3 Schaden
```
Comment (Sqlele) [inline de.js:737]: Same thing as in line 210

File: de.js [hunk 59]
```diff
@@ -717,65 +717,65 @@ server.config.tickRI Hohe Tickrate
...
-server.config.t1Dmg Team 1 Schaden
-server.config.t2Dmg Team 2 Schaden
-server.config.t3Dmg Team 3 Schaden
-server.config.t4Dmg Team 4 Schaden
-server.config.t5Dmg Team 5 Schaden
-server.config.selTeam Team auswählen
-server.config.frFire Friendly Fire
+server.config.nameTeam1 Mannschaft 1 Name
+server.config.nameTeam2 Mannschaft 2 Name
+server.config.nameTeam3 Mannschaft 3 Name
```
Comment (Sqlele) [inline de.js:732]: Same thing as in line 210

File: de.js [hunk 60]
```diff
@@ -534,21 +534,21 @@ windows.controls.interactSec Sekundär Interaktion
...
 windows.controls.specNames Erzwungene Namen umschalten
 windows.controls.specFocus Spielerfokus umschalten
 windows.controls.hidePlayers Verstecke Spieler
-windows.controls.streak0 Abschussserie 1
-windows.controls.streak1 Abschussserie 2
-windows.controls.streak2 Abschussserie 3
-windows.controls.streak3 Abschussserie 4
-windows.controls.streak4 Abschussserie 5
+windows.controls.streak0 Abschusssträhne 1
+windows.controls.streak1 Abschusssträhne 2
```
Comment (Sqlele) [inline de.js:548]: Same thing as in line 64

File: de.js [hunk 61]
```diff
@@ -462,14 +462,14 @@ settings.social.bg Hintergrundbild
 settings.social.bg.paste Hintergrundbild URL
 windows.settings.header Spieleinstellungen
 windows.mail.header Krunker Mail
-windows.servers.header Server Browser
+windows.servers.header Server Suchmaschine
```
Comment (Sqlele) [inline de.js:465]: Same thing as in line 13

**Review by Sqlele — COMMENTED** [review]
Hello, I'm a native german speaker and I wan't to help. Someone should double check his translation and my review to avoid errors

**Review by ShakeAndWiggle20 — APPROVED** [review]
Well done! Good translation.

**Comment by mty22** [comment]
I'm not sure who you are @ShakeAndWiggle20, although this is not being merged until @Sqlele's comments are addressed.

Also, please rebase from `main` as there's merge conflicts.

Will close this as I am treating it as stale. Re-open once you've fixed the merge conflicts.

**Comment by Customsexe** [comment]
> I'm not sure who you are @ShakeAndWiggle20, although this is not being merged until @Sqlele's comments are addressed.
> 
> Also, please rebase from `main` as there's merge conflicts.
> 
> Will close this as I am treating it as stale. Re-open once you've fixed the merge conflicts.

Do i have to recreate it?

**Pull request closed without merging**